app/src/main/java/org/dicekeys/app/extensions/FragmentExtensions.kt

```diff
@@ -103,4 +105,34 @@ fun Fragment.showPopupMenu(view: View, @MenuRes menuRes: Int, fn: ((popupMenu: P
     popup.setOnMenuItemClickListener(listener)
     fn?.invoke(popup)
     popup.show()
+}
+
+fun Fragment.askToCopyToClipboard(content: String, animateView: View? = null){
+    MaterialAlertDialogBuilder(requireContext())
+        .setTitle("Copy to Clipboard?")
```

Comment:
Is capitalizing Clipboard something that is recommended by the platform?

Reply:
nope. changing it.